synapse/api/errors.py

```diff
@@ -210,6 +210,11 @@ def __init__(
     def error_dict(self, config: Optional["HomeServerConfig"]) -> "JsonDict":
         return cs_error(self.msg, self.errcode, **self._additional_fields)
 
+    @property
+    def debug_context(self) -> Optional[str]:
+        """Override this to add debugging context that shouldn't be sent to clients."""
```

Comment:
It might be "overload"; I always confuse those two.

Reply:
Override seems correct.